install/manual_upgrade_charset_utf8.txt

```diff
@@ -0,0 +1,6 @@
+How to change the charset in the database to utf8 from v1.3
+
+    -Open SQL file in /coral/resources/install/protected/update_latin1-to-utf8.sql and replace _DATABASE_NAME_ with your database schema name
```

Comment:
These instructions still mention the old "_DATABASE_NAME_" prefix, which has been removed from install/update scripts in this module. So this line can be removed from the instructions. If this database name prefix is still used anywhere, it should be removed.